source/API/CMakeLists.txt

```diff
@@ -185,9 +186,22 @@ add_custom_command_target(
     ALL
     DEPENDS ${clang_headers_target})
 
+# Copy the swift resource directory.
+add_custom_command_target(
```

Comment:
This bit worries me. @JDevlieghere ?

Reply:
or @adrian-prantl

Reply:
Yeah this could be just an `add_custom_command`. We can change the clang resource directory invocation too, in a separate change.